chunks/lang-java/Dockerfile

```diff
@@ -8,18 +8,18 @@ USER gitpod
 # Dazzle does not rebuild a layer until one of its lines are changed. Increase this counter to rebuild this layer.
 ENV TRIGGER_REBUILD=1
 
+SHELL [ "/bin/bash", "-c" ]
 RUN curl -fsSL "https://get.sdkman.io" | bash \
- && bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh \
-             && sed -i 's/sdkman_selfupdate_enable=true/sdkman_selfupdate_enable=false/g' /home/gitpod/.sdkman/etc/config \
-             && sed -i 's/sdkman_selfupdate_feature=true/sdkman_selfupdate_feature=false/g' /home/gitpod/.sdkman/etc/config \
-             && sdk install java ${JAVA_VERSION} \
-             && sdk install gradle \
-             && sdk install maven \
-             && sdk flush archives \
-             && sdk flush temp \
-             && mkdir /home/gitpod/.m2 \
-             && printf '<settings>\n  <localRepository>/workspace/m2-repository/</localRepository>\n</settings>\n' > /home/gitpod/.m2/settings.xml \
-             && echo 'export SDKMAN_DIR=\"/home/gitpod/.sdkman\"' >> /home/gitpod/.bashrc.d/99-java \
-             && echo '[[ -s \"/home/gitpod/.sdkman/bin/sdkman-init.sh\" ]] && source \"/home/gitpod/.sdkman/bin/sdkman-init.sh\"' >> /home/gitpod/.bashrc.d/99-java"
-# above, we are adding the sdkman init to .bashrc (executing sdkman-init.sh does that), because one is executed on interactive shells, the other for non-interactive shells (e.g. plugin-host)
-ENV GRADLE_USER_HOME=/workspace/.gradle/
+	&& source "$HOME/.sdkman/bin/sdkman-init.sh" \
+	&& sed -i -e 's/sdkman_selfupdate_enable=true/sdkman_selfupdate_enable=false/g' \
+	-e 's/sdkman_selfupdate_feature=true/sdkman_selfupdate_feature=false/g' "$HOME/.sdkman/etc/config" \
+	&& sdk install java ${JAVA_VERSION} \
+	&& sdk install java 17.0.3.fx-zulu \
```

Comment:
Why this PR has so many changes I thought only one line is required: `sdk install java 17.0.3.fx-zulu`?

Reply:
It was me, I had made some commits but it looks like that they were squashed together into the first commit and then force pushed 😅 :
<img width="1089" alt="Screenshot 2022-07-27 at 5 38 24 PM" src="https://user-images.githubusercontent.com/39482679/181237766-f222c657-d89e-4f51-ba8d-b819bb7b17f5.png">

Do you want me to push these for the sake of commit history?
<img width="916" alt="Screenshot 2022-07-27 at 5 50 14 PM" src="https://user-images.githubusercontent.com/39482679/181239875-b0b5ac20-e3ad-42b3-92a7-243a4c9eb0d4.png">

Reply:
> It was me, I had made some commits but it looks like that they were squashed together into the first commit and then force pushed 😅 : <img alt="Screenshot 2022-07-27 at 5 38 24 PM" width="1089" src="https://user-images.githubusercontent.com/39482679/181237766-f222c657-d89e-4f51-ba8d-b819bb7b17f5.png">
> 
> Do you want me to push these for the sake of commit history? Edit: I merged and pushed. <img alt="Screenshot 2022-07-27 at 5 50 14 PM" width="916" src="https://user-images.githubusercontent.com/39482679/181239875-b0b5ac20-e3ad-42b3-92a7-243a4c9eb0d4.png">

@akosyakov and I agreed to rebase the PR (as we do for gitpod's main repo).

Reply:
> @akosyakov and I agreed to rebase the PR and squash the 16 commits to avoid noise (as we do for gitpod's main repo).

Fair enough 👍 
I think we can use the `Squash and merge` button from here on GitHub, that does the same thing a bit differently right? Since you know, force pushing can cause issues on local branches when multiple people are collaborating on the same PR 🙏 
Just my two cents.

Reply:
Can we revert please all noise to the minimal one line change? It should have been very simple and quick PR to unblock our Java users.

Reply:
Sure 👍 
I reverted the extra changes. While adding the new tests I had faced an issue, so I thought to put the fix in here an an one-off thing 😅 
Will create a separate PR for that later 👍 
Sorry for the trouble 🙏

Reply:
Thank you @akosyakov and @axonasif , this is helpful.